_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Squeezing the most out of your pebble"
+date:   2020-04-13 09:00:42
+# categories: community
+---
+
+In the ~6 months since our [last blog post](https://rebble.io/2019/11/02/rebble-rebble-everywhere.html) a lot has happened, both in the pebbleverse and the real world. Yet, despite the ever-increasing gap between the pebble shutdown and today, still new people are signing up with rebble to bring their beloved smartwatch back to life!
```

Comment:
capitalize Pebbleverse, Pebble

Reply:
Maybe good to start off 'Hi there!  Will Murphy here (some of you might know me as Will0 on Discord)' or something, since we don't have blog attribution.